pkgs/build-support/rust/build-rust-crate/configure-crate.nix

```diff
@@ -13,7 +13,7 @@
 , crateRenames
 , crateVersion
 , extraLinkFlags
-, extraRustcOpts
+, extraRustcOptsForBuild
```

Comment:
I find it a bit confusing that extraRustcOpsForBuild is used in the configuration phase and extraRustcOps is used in the build phase.
Would it make sense to rename this to something like extracRustcOpsForConf ? Not sure, just thinking out loud.

Reply:
I see, that is confusing. the "for build" is somewhat standard in that it means the "build platform", but not so standard in rust land.